src/content/en/fundamentals/engage-and-retain/push-notifications/sending-messages.markdown

```diff
@@ -3,93 +3,343 @@ layout: shared/narrow
 title: "Sending messages"
 description: "There are two servers involved in sending a message: your server and a third party messaging server. You keep track of who to send messages to. The third party server handles the routing."
 published_on: 2016-07-01
-updated_on: 2016-07-01
+updated_on: 2016-08-05
 order: 30
 translation_priority: 0
 authors:
   - josephmedley
 ---
 
-<p class="intro"> It's easy to assume that sending messages must involve
-building a complex messaging system on the server. That's not quite right. There
-are actually two servers involved in sending a message: your server and a third
-party messaging server. You keep track of two things: recipients and recipient-
-specific endpoints on the messaging server. The messaging server handles the
-routing. </p>
+<p class="intro">
+It's easy to assume that sending messages must involve building a complex 
```

Comment:
Just mentioning complexity, makes me assume it's complex, no matter what follows that statement ;)